File: Java/tarifa.java
import java.util.Scanner;
public class Main {
  public static void main(String[] args) {
    Scanner kb = new Scanner(System.in);
    int x = kb.nextInt();
    int mbLeft = 0;
    int tc = kb.nextInt();
    for (int i = 0; i<tc; i++){
      int mbUsed = kb.nextInt();
      mbLeft+=x-mbUsed;
    }
    System.out.println(mbLeft+x);
  }
}

File: C++/tarifa.cpp
#include <iostream>

int main() {
  int x;
  std::cin>>x;
  int mbLeft = 0;
  int tc;
  std::cin>>tc;
  for (int i = 0; i<tc; i++){
    int mbUsed;
    std::cin>>mbUsed;
    mbLeft+=x-mbUsed;
  }
  std::cout<<mbLeft+x<<std::endl;
}
